src/createNumbersGenerator.js

```diff
@@ -30,7 +30,21 @@
  * @param {number} max - (optional)
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let currentNumber = 0;
+
+  return () => {
+    let arr = [ ...numbers ];
```

Comment:
что такое arr?
```suggestion
    let arr = [...numbers];
```
отступы не нужны